lxd/auth/drivers/tls.go

```diff
@@ -161,6 +169,13 @@ func (t *tls) GetPermissionChecker(ctx context.Context, entitlement auth.Entitle
 			return false
 		}
 
+		// Allow identities (certificates if using this authorizer) to view, edit, delete themselves.
+		// Allow identities (certificates if using this authorizer) to view themselves.
```

Comment:
I am a bit confused by these two similar comments. In my understanding just the second line is needed here since the permissions for edition and deletion are explained below, what do you think?

Reply:
Ah whoops I didn't mean to leave the first comment in there!